docs/articles/vs-test-adapter/Tips-And-Tricks.md

```diff
@@ -57,6 +57,10 @@ See <https://github.com/nunit/nunit3-vs-adapter/blob/8a9b8a38b7f808a4a78598542dd
 
 ### Details
 
+#### DefaultTestNamePattern
+
+The default format used to provide test names, expressed as an NUnit [Template Based Test Name Pattern](https://docs.nunit.org/articles/nunit/running-tests/Template-Based-Test-Naming.html).
```

Comment:
```suggestion
The default format used to provide test names, expressed as an NUnit [Template Based Test Name Pattern](xref:TemplateBasedTestNaming).
```

Suggestion to use an xref in case we move the file in the future and the location changes.